doc/content/reference/api/_index.md

```diff
@@ -4,3 +4,13 @@ description: ""
 ---
 
 This is the reference for the gRPC and HTTP APIs that {{% tts %}} exposes.
+
+<!--more-->
+
+## HTTP Queries
+
+Additional fields may be specified in HTTP requests by appending them as query string parameters. For example, to request the `name`, `description`, and `locations` of devices in an `EndDeviceRegistry.Get` request, add these fields to the `field_mask` field:
```

Comment:
I think we should write a little more about what field masks are, and how we use them in our APIs (because we use them on Get/List, but also on Update RPCs).

See also https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.FieldMask

Reply:
Also `EndDeviceRegistry` doesn't exist; they're separate per component

Reply:
> Also `EndDeviceRegistry` doesn't exist; they're separate per component

Ah I see what you mean, but the IS `EndDeviceRegistry` is just called `EndDeviceRegistry` https://thethingsstack.io/reference/api/end_device/#the-enddeviceregistry-service

Reply:
You're right!